src/common/api-interface.ts

```diff
@@ -145,6 +146,11 @@ export interface IVersionInfo {
     searchApiVer: string;
 }
 
+export interface ICPUUsage {
+    percentCPUUsage: number;
```

Comment:
and same here, percentCpuUsage

Reply:
Will keep because of https://www.electronjs.org/docs/api/structures/cpu-usage